frontend/app/admin/src/views/settings/organization-records/EditOrganizationRecordSettings.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+    <div>
```

Comment:
Waarom is deze component een duplicate van `frontend/shared/components/src/records/components/EditRecordCategoriesBox.vue`, wat is het verschil met die component en waarom wordt die niet gebruikt?